packages/graphql-types-generator/src/flow-modern/__tests__/codeGeneration.ts

```diff
@@ -18,6 +18,16 @@ function compile(
   return compileToIR(schema, document, options);
 }
 
+function matchOutputSnapshot(output: any): void {
+  expect(output).toBeInstanceOf(Object);
+
+  Object.keys(output).forEach(filePath => {
+    // convert the windows style path to unix style
+    expect(filePath.replace(/\\/g, "/")).toMatchSnapshot();
+    expect(output[filePath]).toMatchSnapshot();
```

Comment:
Do we have the same issue with the modelgen tests?

Reply:
I did not see any failures in modelgen tests.